README.md

```diff
@@ -319,6 +319,10 @@ Copy **https://pitangui.amazon.com/api/skill/link/YOURVENDORID** or **https://la
 ```
 ACCESS_KEY_ID=<your aws account access key ID>
```

Comment:
This is only required for local development right?
Just add that in the comment as well and make it optional.